front/src/common/Map/Buttons/MapButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { type ReactNode } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import Tipped from 'common/Tipped';
+
+type MapButtonProps = {
+  onClick: () => void;
+  isNewButtons: boolean;
```

Comment:
nit: it's the prop of 1 button
```suggestion
  isNewButton: boolean;
```

Reply:
Done